### PR TITLE
Fix Median and query errors

### PR DESCRIPTION
**Pull request by danielkmariam**

### CODE REVIEW

File: src/Command/MaximumHourly.php [hunk 1]
```diff
@@ -63,6 +63,6 @@ protected function execute(InputInterface $input, OutputInterface $output)
      */
     protected function changeAmPmTo24Hr(InputInterface $input)
     {
-        return DATE("G", STRTOTIME($input->getArgument('time'))) + 1;
+        return DATE("G", STRTOTIME($input->getArgument('time')));
```
Comment (ThePixelDeveloper) [inline src/Command/MaximumHourly.php:66]: Should be lowercased date.

**Review by ThePixelDeveloper — COMMENTED** [review]
Fix casing and good to merge 👍

File: src/Command/MaximumHourly.php (hunk 1 above)
Comment (danielkmariam) [reply src/Command/MaximumHourly.php:66]: lowe cased DATE and STRTOTIME

Comment (danielkmariam) [reply src/Command/MaximumHourly.php:66]: PR commented addressed, merging PR

**Pull request merged**